velox/docs/functions/spark/bitwise.rst

```diff
@@ -12,6 +12,17 @@ Bitwise Functions
     Returns the bitwise OR of ``x`` and ``y`` in 2's complement representation.
     Corresponds to Spark's operator ``^``.
 
+.. spark:function:: bit_count(x) -> integer
+
+    Returns the number of bits that are set in the argument ``x`` as an unsigned 64-bit integer,
+    or NULL if the argument is NULL.
+
+.. spark:function:: bit_get(x, pos) -> byte
```

Comment:
Let's clarify what types are supported.

`Supported types of 'x' are: TINYINT, SMALLINT, INTEGER and BIGINT. Supported types of 'pos' are: INTEGER.`